examples/babel/README.md

```
@@ -0,0 +1,48 @@
+This exists to pin down the `catalogues: 'inline'` default: `babel-plugin-saykit` on its
+own has to rewrite the macros _and_ resolve the catalogue imports, with no other tool
+involved. Every other example runs the plugin alongside a bundler integration, so this is
+the only one that would notice if that stopped being true.
+
+## Run
+
+```sh
+pnpm start
+```
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)